web/posts/2018-07-29-how-i-use-stuff-day-one.markdown

```diff
@@ -0,0 +1,184 @@
+thoughts and feelings in a place that I knew was private and non-judgmental.
+Since then, my usage has largely stayed the same, although I've been able to
+be much more consistent in the past year. I've also added a number of smaller
+semi-automated entries to my practice.
+
+[quitting]: /quitting
+
+## The Workflow
+As of this writing, I have a grand total of 2,144 entries stored in Day One.
+Those entries are split up across a few different journals:
+
+### Journal (1,191 Entries)
+This is my "main" journal, and is where all of my free-form writing goes. I
+don't have any hard and fast rules about this journal, and it's littered with
+photos of graffiti I see while traveling, dumb jokes that I don't feel like
+posting publicly, and longer form entries. I try to write at _least_ one of
+those long-form entries in here every day.
+
+The format of what I count as my daily long-form entry has gone through
+multiple revisions over time. When I first started keeping a journal, I wrote
+multiple entries each day, each with a specific topic. I wrote one about my
+workday, one for my personal thoughts/feelings, and one for each activity I
+did. That pace wasn’t maintainable, and it became exhausting. I eventually
+whittled that down to just writing a single large daily overview entry, but
+got a bit burned out on that as well.
+
+More recently, I've settled into just writing _something_ in here every day. I
+try hard to not judge myself on length or quality of the entry. Instead, I
+just focus on getting something written down. I've moved away from entries
+that are glorified lists of things I did during the day, and I focus more on
+writing about what I _feel_. One of the biggest benefits of journaling for me
+is being able to look back at where I was years ago and see how things have
+changed. The list-style entries have limited interest as time goes on, but the
+entries that center around my feelings and my general outlook are evergreen.
+
+### Daily Q&A (651 Entries)
+When I started my journaling practice, one of the things I wanted to replicate
+the most was the [Daily Q&A Journal][physical-q-a-journal] that I bought for
+my wife nearly 4 years ago. Answering the same question on the same day year
+after year is a great way to watch your responses change over time. However,
+carrying around a physical book for that was a non-starter for me. Thankfully,
+Day One has an excellent "On This Day" feature, where it shows you all the
+entries you wrote on the current day of past years. So all that was left was
+to add prompts.
+
+[physical-q-a-journal]: https://www.amazon.com/Day-5-Year-Journal-Potter-Style/dp/0307719774
+
+After painstakingly transcribing every question in the above physical book and
+translating them into JSON, I was able to publish [a super small web app to
+spit them back out at me][daily-prompt] ([the source is public if you're a
+nerd][daily-prompt-source]). I have this link bookmarked in my Favorites for
```

Comment:
Heh. This might be a little confrontational. Maybe `s/nerd/Swift-curious` or just `curious`, `interested`, `if you want to replicate this yourself`?

Reply:
Also: `bookmarked in your Favorites for quick access`. The `in your Favorites for quick access` seems kind of obvious? Why/where else would it be bookmarked?